include/bluetooth/hci.h

```diff
@@ -815,6 +815,15 @@ struct bt_hci_cp_le_set_random_address {
 #define BT_HCI_ADV_DIRECT_IND_LOW_DUTY          0x04
 #define BT_HCI_ADV_SCAN_RSP                     0x04
 
+#define BT_LE_ADV_INTERVAL_MIN                  0x0020
+#define BT_LE_ADV_INTERVAL_MAX                  0x4000
+#define BT_LE_ADV_INTERVAL_DEFAULT              0x0800
+
+#define BT_LE_ADV_CHAN_MAP_CH37                 0x01
+#define BT_LE_ADV_CHAN_MAP_CH38                 0x02
+#define BT_LE_ADV_CHAN_MAP_CH39                 0x04
```

Comment:
For consistency I would name them CHAN_37/38/39 (bluetooth.h has this).

Reply:
fixed